packages/platform-sdk-btc-indexer/src/database.ts

```diff
@@ -134,20 +119,25 @@ export class Database {
 	 * @param {*} transaction
 	 * @memberof Database
 	 */
-	private storeTransaction(transaction): void {
+	private async storeTransaction(transaction): Promise<any> {
 		const amount: BigNumber = getAmount(transaction);
 		const vouts: VOut[] = getVOuts(transaction);
 		const vIns = getVIns(transaction);
```

Comment:
VIn and VOut naming perhaps a big obscure for the uninitiated. I wouldn't hesitate making types & variable names a bit more explicit here

Reply:
Not really sure either else to call them or whether to actually do it. That's the name that comes out of the BTC api. So even if unfamiliar to the uninitiated, I can't think of a more accurate name. All open to ideas though...

Reply:
`inputs` and `outputs` should describe them the same way but would be less technical.